about.html

```diff
@@ -1 +1,187 @@
+              <strong>Hometown:</strong>
+            </div>
+            <div class="col-xs-8">
+              Godrick's Hollow, England
+            </div>
+          </div>
+          <div class="row extra-line-height">
+            <div class="col-xs-4">
+              <strong>Currently lives:</strong>
+            </div>
+            <div class="col-xs-8">
+              Godrick's Hollow, England
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-xs-12">
+              <h3>Contact Information</h3>
+            </div>
+          </div>
+          <div class="row extra-line-height">
+            <div class="col-xs-4">
+              <strong>Email:</strong>
+            </div>
+            <div class="col-xs-8">
+              harry_potter@hogwarts.edu
+            </div>
+          </div>
+          <div class="row extra-line-height">
+            <div class="col-xs-4">
+              <strong>Telephone:</strong>
+            </div>
+            <div class="col-xs-8">
+              555-123-4567
+            </div>
+          </div>
+        </div>
+        <div class="col-md-5 col-md-offset-1">
+          <div class="row">
+            <div class="col-md-12">
+              <h3>Words to live by</h3>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-md-12">
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+              tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+              quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+              consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+              cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+              proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-md-12">
+              <h3>About Me</h3>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-md-12">
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+              tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+              quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+              consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+              cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+              proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
```

Comment:
This was a well organized HTML file, but I'd add more comments in the future to help discern components.